<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatGPTClass</class>
 <widget class="QWidget" name="ChatGPTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatGPT</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextEdit
{
    background-color: #F5F5F5;
    color: #333333;
    border: 1px solid #CCCCCC;
    border-radius: 4px;
    padding: 4px;
}

QTextEdit:hover 
{
    background-color: #EDEDED;
}

QTextEdit:focus
{
    border: 1px solid #3399FF;
    outline: none;
}
QWidget 
{
    background-color: #f2f2f2;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 5px;
	font: 10pt &quot;Microsoft YaHei UI&quot;;
}


QToolButton 
{
	font: 10pt &quot;Microsoft YaHei UI&quot;;
    background-color: #f2f2f2;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 5px;
}
QToolButton:hover 
{
    background-color: #e6e6e6;
}
QToolButton:pressed 
{
    background-color: #d9d9d9;
}

QGroupBox 
{
    border: 1px solid #cccccc;
    border-radius: 4px;
    margin-top: 10px;
}
QGroupBox:title 
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5px;
    font-weight: bold;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnTop">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnWriteFile">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSetting">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnClear">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSend">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnGitHub">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>ChatGPT</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTextEdit" name="textEditGPT">
           <property name="styleSheet">
            <string notr="true">	font: 700 11pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="autoFormatting">
            <set>QTextEdit::AutoAll</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>询问ChatGPT</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEditUser">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">	font: 700 11pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ChatGPT.qrc"/>
 </resources>
 <connections/>
</ui>
